client/src/app/shared/components/chip/chip.component.scss

```diff
@@ -0,0 +1,59 @@
+/**
+ * For further information, see https://css-tricks.com/switch-font-color-for-different-backgrounds-with-css/
+ */
+:host {
+    --os-chip-red: 0;
+    --os-chip-green: 0;
+    --os-chip-blue: 0;
+
+    /**
+     * The threshold at which colors are considered "light".
+     * Range: decimals from 0 to 1,
+     * recommended 0.5 - 0.6
+     */
+    --os-chip-threshold: 0.5;
+    /**
+     * The threshold at which a darker border will be applied.
+     * Range: decimals from 0 to 1,
+     * recommended 0.8+
+     */
+    --os-chip-border-threshold: 0.8;
+}
+
+.os-chip {
```

Comment:
this is very nice. Didn't expect it to be already implemented. Well done!

Reply:
Thanks!